### ISSUE TITLE
LineString disappears

### ISSUE THREAD

**Issue by stemicz** [pauldendulk/Mapsui#58]
Hi,
I found a route by Windows.Services.Maps.MapRouteFinder  and put coordinates of points to Feature with LineString. When I add layer with LineString, almost everything was ok. But when resolution is smaller than 7, path just disappears. 
Thank you

**Comment by pauldendulk** [comment]
What plaftorm are you using? (WPF, Silverlight, iOS, Android)
Does the with of the line stay the same when you zoom in and out?
You say the resolution is smaller, so this means when you view more of the map the line disappears?

**Comment by stemicz** [comment]
I am trying to implement to UWP app. And the width is same when I zoom. And smaller resolution I mean when I zoom in. On the first picture is a blue thin line. On the second one is MyMap.Viewport.Resolution smaller (exactly set to 6) and line disappeared.
![bigger resolution](https://cloud.githubusercontent.com/assets/15275632/10813462/8c2ec564-7e1f-11e5-85c5-d7c55eee253d.PNG)
![smaller resolution](https://cloud.githubusercontent.com/assets/15275632/10813461/8c2e6e7a-7e1f-11e5-8158-b5d6d0689acb.PNG)

**Comment by pauldendulk** [comment]
I can remember a similar problem when a polygon completely disappeared when we zoomed in (you had to zoom quite deep though). When we tracked down the problem seemed to be with a limitation of the Windows 10 'wpf' renderer. It seemed the result of some performance optimization. Perhaps this is a similar problem. This would mean the UWP renderer is useless unfortunately :(

The probleem also seems to be releated to working with very small numbers in the transformations. Perhaps a workaround could be implemented in the renderer that way. Unfortunately I do not have time for this in the short run.

For the long run we would like to replace all UIElement based rendering with hardware accelerated renderers. For UWP we would like to use: https://github.com/Microsoft/win2d. Some experiments were promising.
